plugins/plugin-site/src/components/Plugin.jsx

```diff
@@ -36,7 +36,7 @@ function Plugin({plugin: {name, title, stats, labels, excerpt, developers, build
             </div>
             <div className="Plugin--InstallsContainer">
                 {'Installs:  '}
```

Comment:
Does this need a new label, e.g. "Popularity" or "Install Stats"? Or just "Installed on " / "Installed by"? Is that a complete sentence?

Reply:
Good point. In the search page we could squeze in "installed on X% instances"
![image](https://github.com/jenkins-infra/plugin-site/assets/1105305/541c75ba-540d-4b67-bb41-577db026671a)
I'll check if we can do that somehow also for plugin pages.

Reply:
Grammar-wise, I think it would be "of instances"?

Reply:
Would "Install rate" make sense?  @MarkEWaite thoughts?

Reply:
> Would "Install rate" make sense? @MarkEWaite thoughts?

I'm accustomed to "rate" having a stronger relationship to time than this measure seems to have.  

I like "Installed on 35% of instances" as much clearer than "105000".

Reply:
So I changed it to "used by" in search screen where there is less space and "installed on" in the plugin page where a line break is possible. If anyone has a preference for some other wording I can change it.

Reply:
> <img width="361" alt="Screenshot 2023-09-22 at 14 34 46" src="https://github.com/jenkins-infra/plugin-site/assets/1831569/c332d814-a822-4f38-8d08-794afbe4524a">

Fits nicely (fake data) 👍